package.json

```diff
@@ -6,6 +6,8 @@
   "scripts": {
     "start": "vite --host --port=8100",
     "dev": "vite",
+    "dev:external": "ionic cap run android -l --external",
+    "dev:device": "ionic cap run android -l --external --target=$DEVICE",
```

Comment:
is the device one rely useful? Just a thought, but I'm totally fine keeping it.

Reply:
The only purpose of the device script is to skip the device choice when running the command. I have found that it dramatically speeds up development

Reply:
Sounds good!

Reply:
might want to put in the README that one needs to set that env variable in order to use it.